Java/AutoML/README.md

```diff
@@ -0,0 +1,23 @@
+
+
+# Java工程 AutoML
+
+分为下面几个模块
+> automl-cms-api    : some application
+> automl-runner     : some mq and rpc
+> automl-component  : provide cms and component to service
+> automl-sdk        : pause
+
+## 1. init dataset Service
+> 区分test和prod环境
+> resources/application.yml/ 中
+>  profiles:
+     active: test 控制是从那个DB获取数据
+>   prod : automl
+>   test : automl-test
+
+> 执行完sql后, 点击cms.api/application启动容器.
+> 浏览器访问: localhost:8081/dataset
+![请添加图片描述](https://img-blog.csdnimg.cn/90c6ffe8aad942819037863bf06e5cb2.png)
```

Comment:
Please correct everything since it's a formal commit instead of investigation.

Reply:
ok, i fixed  from [[fix Java readme.md]](https://github.com/ml-for-good/automl/pull/71/commits/1b86c7ec6d5be467560b13e6bce917b2474c60b9)